lstchain/reco/r0_to_dl1.py

```diff
@@ -555,12 +562,12 @@ def r0_to_dl1(
                     dl1_container.trigger_type = event.r0.tel[telescope_id].trigger_type
 
                     # FIXME: no need to read telescope characteristics like foclen for every event!
-                    foclen = event.inst.subarray.tel[telescope_id].optics.equivalent_focal_length
-                    mirror_area = u.Quantity(event.inst.subarray.tel[telescope_id].optics.mirror_area, u.m ** 2)
+                    foclen = subarray.tel[telescope_id].optics.equivalent_focal_length
+                    mirror_area = u.Quantity(subarray.tel[telescope_id].optics.mirror_area, u.m ** 2)
                     width = np.rad2deg(np.arctan2(dl1_container.width, foclen))
                     length = np.rad2deg(np.arctan2(dl1_container.length, foclen))
-                    dl1_container.width = width.value
-                    dl1_container.length = length.value
+                    dl1_container.width = width
```

Comment:
this has `deg` units, can we include that in the container?

Reply:
already the case

Reply:
ok so we were writing them before in `deg`, but the unit in the containers was wrong?